<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConverterWindow</class>
 <widget class="QMainWindow" name="ConverterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ConverterWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>8</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="handleWidth">
       <number>2</number>
      </property>
      <widget class="QWidget" name="widget_3" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>64</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>128</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>1</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="SourceLabel">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Input</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="SourceDirectory"/>
          </item>
          <item>
           <widget class="QPushButton" name="SourceFinder">
            <property name="maximumSize">
             <size>
              <width>24</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="widget_5" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <property name="spacing">
            <number>4</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QCheckBox" name="SourceIsRecursive">
             <property name="text">
              <string>Read recursively</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_8">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>233</width>
               <height>18</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Encode</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="SourceEncodingType">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>16</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>Shift-JIS</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>EUC-JP</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>UTF-8</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="DestLabel">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Output</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="DestDirectory"/>
          </item>
          <item>
           <widget class="QPushButton" name="DestFinder">
            <property name="maximumSize">
             <size>
              <width>24</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <property name="spacing">
            <number>4</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>233</width>
               <height>17</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Encode</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="DestEncodingType">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>16</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>Shift-JIS</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>EUC-JP</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>UTF-8</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="Line" name="line_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>3</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Compression settings</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_7" native="true">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>24</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>2</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="MelcepLabel">
               <property name="text">
                <string>MelCepstrum Dimension</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="MelcepstrumDim">
               <property name="minimum">
                <number>32</number>
               </property>
               <property name="maximum">
                <number>1024</number>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="VorbisLabel">
               <property name="text">
                <string>OggVorbis bitrate[kbps]</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="VorbisBitrate">
               <property name="minimum">
                <number>32</number>
               </property>
               <property name="maximum">
                <number>512</number>
               </property>
               <property name="value">
                <number>64</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="EnableNormalize">
            <property name="text">
             <string>Normalize oto.ini file</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_6" native="true">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>24</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="spacing">
              <number>2</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>49</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="PreLabel">
               <property name="text">
                <string>     PreUtterance [msec]</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="PrePronounce">
               <property name="maximum">
                <number>2000</number>
               </property>
               <property name="value">
                <number>250</number>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>49</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="PhonemeLabel">
               <property name="text">
                <string>Phoneme Length[msec]</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="PhonemeLength">
               <property name="maximum">
                <number>9999</number>
               </property>
               <property name="value">
                <number>500</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_8" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Analysis settings</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_9" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <property name="spacing">
                 <number>2</number>
                </property>
                <property name="margin">
                 <number>0</number>
                </property>
                <item>
                 <spacer name="horizontalSpacer_10">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Frame Shift[msec]</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="AnalysisFramePeriod">
                  <property name="minimum">
                   <double>1.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>10.000000000000000</double>
                  </property>
                  <property name="value">
                   <double>2.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>How many threads</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="ThreadNumber">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>6</number>
                  </property>
                  <property name="value">
                   <number>1</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widget_4" native="true">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>48</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>334</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="ConvertButton">
          <property name="text">
           <string>Convert</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="ExitButton">
          <property name="text">
           <string>Exit</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="AnalysisProgress">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="format">
       <string>%v/%m</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>474</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>ExitButton</sender>
   <signal>clicked()</signal>
   <receiver>ConverterWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>422</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>EnableNormalize</sender>
   <signal>clicked()</signal>
   <receiver>ConverterWindow</receiver>
   <slot>normalizeEnabledChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>37</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SourceFinder</sender>
   <signal>clicked()</signal>
   <receiver>ConverterWindow</receiver>
   <slot>sourceFinderClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DestFinder</sender>
   <signal>clicked()</signal>
   <receiver>ConverterWindow</receiver>
   <slot>destFinderClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ConvertButton</sender>
   <signal>clicked()</signal>
   <receiver>ConverterWindow</receiver>
   <slot>convertButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>normalizeEnabledChanged()</slot>
  <slot>sourceFinderClicked()</slot>
  <slot>destFinderClicked()</slot>
  <slot>convertButtonClicked()</slot>
 </slots>
</ui>
